web/src/components/FiltersForm.res

```diff
@@ -2,7 +2,8 @@ open Prelude
 
 // The definition of a filter:
 module Filter = {
-  type kind = Text | Date | Choice(list<string>)
+  type choiceType = Free(list<string>) | RelativeDate(list<string>)
```

Comment:
Perhaps `Keywords` or `TextList` instead of `Free`.
And perhaps add an 's' to `RelativeDates`.